packages/app-builder-lib/src/electron/ElectronFramework.ts

```diff
@@ -97,27 +97,29 @@ async function removeUnusedLanguagesIfNeeded(options: BeforeCopyExtraFilesOption
     return
   }
 
-  const { dir, langFileExt } = getLocalesConfig(options)
+  const { dirs, langFileExt } = getLocalesConfig(options)
   // noinspection SpellCheckingInspection
-  await asyncPool(MAX_FILE_REQUESTS, await readdir(dir), async file => {
-    if (!file.endsWith(langFileExt)) {
-      return
-    }
+  const deletedFiles = async (dir: string) => {
+    await asyncPool(MAX_FILE_REQUESTS, await readdir(dir), async file => {
+      if (path.extname(file) !== langFileExt) {
+        return
+      }
 
-    const language = file.substring(0, file.length - langFileExt.length)
-    if (!wantedLanguages.includes(language)) {
-      return fs.rm(path.join(dir, file), { recursive: true, force: true })
-    }
-    return
-  })
+      const language = path.basename(file, langFileExt)
```

Comment:
So I just double checked in a js playground, and in order to use `basename` with ext, we need to in clude a `.`
```
`.${langFileExt}`
```

Reply:
Yes. I have added `.`